HITC-Py/setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import setup, find_packages
+import sys, os
+
+version = '0.1'
+
+setup(name='HITC-Py',
+      version=version,
+      description="Python client for a running HITC API",
+      long_description="""\
+Python client for running HITC API""",
+      classifiers=[], # Get strings from http://pypi.python.org/pypi?%3Aaction=list_classifiers
+      keywords='api',
+      author='Jonathan Mackenzie',
+      author_email='jonmac1@gmail.com',
+      url='http://github.com/nupic-community/hitc-py',
+      license='AGPLv3',
```

Comment:
I don't think there is any reason to make this AGPLv3 because it's really just a REST client and we're just exchanging JSON. I prefer MIT or BSD 3-clause.